src/merkleblock.cpp

```diff
@@ -5,6 +5,7 @@
 
 #include "merkleblock.h"
 
+#include "consensus/consensus.h"
 #include "hash.h"
 #include "primitives/block.h" // for MAX_BLOCK_SIZE
```

Comment:
If MAX_BLOCK_SIZE really should go into consensus.h then you need to remove this comment (and perhaps we have that also in other places).